src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,63 @@
+package duke.storage;
+
+import duke.task.TaskList;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.OutputStream;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+
+public class Storage {
+    private final String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public void saveTasks(TaskList tasks) {
+        try {
+            OutputStream output = new FileOutputStream("./data.txt");
```

Comment:
I enjoyed the use of serialisation to store and retrieve the tasks without the need for manual input and output 👍

Reply:
thanks